runtime/services/lightd/effects.js

```diff
@@ -50,6 +50,17 @@ var path = require('path')
 
 var SYSTEM_MEDIA_SOURCE = '/opt/media/'
 
+var globalAlphaFactor = 1
+
+function applyAlphaFactor (alpha) {
+  if (alpha !== undefined && typeof a === 'number' && alpha >= 0 && alpha <= 1) {
```

Comment:
`typeof a === 'number'` Where is the variable `a` defined? I guess you spelled wrong.

Reply:
Should be `alpha` I guess, lol.

Reply:
fixed, stupid error, :(